DabraHalleloEndaAbunaSamuel/DHAS022.xml

```diff
@@ -0,0 +1,148 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="DHAS022" xml:lang="en" type="mss">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Fragments of two manuscripts</title>
```

Comment:
It is enigmatic, how a manuscript can be "fragments of two manuscripts" having only on ms_item. Is it possible to explain better in condition or ms_i1?

Reply:
addressed.